dev/com.ibm.websphere.org.eclipse.microprofile.reactive.streams.operators.1.0/bnd.bnd

```diff
@@ -0,0 +1,48 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+# IBM Corporation - initial API and implementation
+# Add to Include Resource when present
+# We will add this in when it is in maven central
+# @${repo;org.eclipse.microprofile.reactive.streams:microprofile-reactive-streams-operators;1.0;EXACT}
+# and remove the src directory
+# We removed 
+# com.ibm.websphere.org.reactivestreams.reactive-streams.1.0.2;version=latest
+# from Export-package to remove the org reactive streams classes from the
+# jar
+# com.ibm.websphere.org.reactivestreams.reactive-streams.1.0.2;version=latest
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-SymbolicName: com.ibm.websphere.org.eclipse.microprofile.reactive.streams.operators.1.0; singleton:=true
+
+Import-Package: \
+
+Export-Package: \
+  org.eclipse.microprofile.reactive.streams;version=1.0, \
+  org.eclipse.microprofile.reactive.streams.spi;version=1.0, \
+
+Include-Resource: \
+
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+src: api/src
```

Comment:
the target, spec and tck folders aren't needed in this project so `api/src` can be moved back down to just `src`